docs/user-guide.rst

```diff
@@ -55,6 +55,12 @@ HTTP verb:
 The :ref:`request_data` section covers sending other kinds of requests data,
 including JSON, files, and binary data.
 
+.. note:: For quick scripts and experiments you can also use a top-level ``urllib3.request()``.
+    It uses a module-global ``DEFAULT_POOL`` ``PoolManager`` instance.
+    Because of that, its side effects could be shared across dependencies relying on it.
+    To avoid side effects, create a new ``PoolManager`` instance and use it instead.
+    In addition, the method skips the low level ``**urlopen_kw`` keyword arguments. It is not recommended for use in production.
```

Comment:
We can remove the note about not being able to use in production. It's production capable given you're okay with the above constraints.

Reply:
At the very least, if people find scenarios where this breaks "in production" in unexpected ways then I'd like to consider it a bug.

Reply:
Removed the recommendation to not use it in production.

Reply:
Also added a note about system CA certificates being loaded by default.